nicsdru_unity_theme.libraries.yml

```diff
@@ -73,3 +73,9 @@ eu-cookie-compliance:
   css:
     state:
       css/4_state/block/eu-cookie-compliance.css: { minified: true }
+
+autocomplete:
+  version: VERSION
```

Comment:
Not sure you'll want this in here as it'll cache this to the Drupal version of core .